docs/helm-chart/keda.rst

```diff
@@ -24,27 +24,23 @@ KEDA stands for Kubernetes Event Driven Autoscaling.
 `KEDA <https://github.com/kedacore/keda>`__ is a custom controller that
 allows users to create custom bindings to the Kubernetes `Horizontal Pod
 Autoscaler <https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/>`__.
-We have built scalers that allows users to create scalers based on
```

Comment:
i removed this sentence because if i recall correctly the "we" is astronomer, so in this repo it doesn't really have the same meaning and i don't think the statement belongs.  but it's also sortof beside the point.

Reply:
Exactly it me and Daniel, so not only Astronomer. Also Airflow is on KEDA users page.